examples/seq2seq/run_trans.py

```diff
@@ -217,6 +218,15 @@ def main():
     else:
         model_args, data_args, training_args = parser.parse_args_into_dataclasses()
 
+    # @patrickvonplaten|@stas00 picked this up from
+    #   https://github.com/huggingface/transformers/issues/10164#issuecomment-779293173
+    #   https://github.com/huggingface/transformers/pull/10611#discussion_r590595798
+    # but unclear for me if I should restrict like this or just `.startswith("t5")`?
+    if data_args.source_prefix is None and model_args.model_name_or_path in ["t5-small", "t5-base", "t5-large", "t5-3b", "t5-11b"]:
+        src = lc.Language(data_args.source_lang).display_name()
+        tgt = lc.Language(data_args.target_lang).display_name()
+        data_args.source_prefix = f"translate {src} to {tgt}: "
```

Comment:
and so the instructions should be updated to remove the `--source_prefix` command then, right?

Or do we need to leave it for some custom user models?

Reply:
yes you're right I forgot to remove it from the docs

and yes currently I'm keeping the parameter and only do the automatic setting if it's not present, so users can override it - but I'm not confident this is the best: what do we favour in these scripts, simpler-tweakable code vs all-terrain-ready-to-use scripts? 
my understanding through this work is that we want the scripts to be the latter, hence my original choice.
if the former I'd remove the `source_prefix` arg altogether

Reply:
Just to clarify further and to give a bit of context, the example scripts are made to be very simple and tweakable, rather than all-purpose. This has been the decision for a while now, and we do not accept requests to add features to example scripts anymore, as that would hinder understandability.

Reply:
Agree with @LysandreJik here. This again is very T5 specific. Also we add the `lc` library only for T5 here - don't think it's worth adding a new dependency for just 5 checkpoints.